ourjapanlife/findadoc-server
# Fix missing credentials for Firestore

Resolves #198


# What changed
1. added dotenv package to load environment variables into process.env
2. added `firebaseServiceAccountKey.json` to `gitignore` to keep `private_key` private
3. removed unused imports from `database.ts`
4. Changed `credentials` variable to point to `firebaseServiceAccountKey.json`


# Testing instructions
1. Run `yarn install` to update dependencies.
2. Make sure to have a private key file generated by Google Firestore.
3. Add the file to root directory and rename to `firebaseServiceAccountKey.json`.
4. Create an environment variable inside of `.env` by adding the line `SERVICE_ACCOUNT_PATH=./firebaseServiceAccountKey.json`.
5. Run `yarn dev` and confirm server is working.
